google/cloud/compute/v1/BUILD.bazel

```diff
@@ -217,6 +217,55 @@ java_gapic_assembly_gradle_pkg(
     ],
 )
 
+##############################################################################
+# Go
+##############################################################################
+load(
+    "@com_google_googleapis_imports//:imports.bzl",
+    "go_gapic_assembly_pkg",
+    "go_gapic_library",
+    "go_proto_library",
+    "go_test",
+)
+
+go_proto_library(
+    name = "compute_go_proto",
+    importpath = "google.golang.org/genproto/googleapis/cloud/compute/v1",
+    protos = [":compute_proto"],
+    deps = [
+        "//google/api:annotations_go_proto",
+    ],
+)
+
+go_gapic_library(
+    name = "compute_go_gapic",
+    srcs = [":compute_proto_with_info"],
+    grpc_service_config = "compute_grpc_service_config.json",
+    importpath = "cloud.google.com/go/compute/apiv1;compute",
+    service_yaml = "compute_v1.yaml",
+    transport = ["rest"],
+    deps = [
+        ":compute_go_proto",
+    ],
+)
+
+go_test(
+    name = "compute_go_gapic_test",
+    srcs = [":compute_go_gapic_srcjar_test"],
+    embed = [":compute_go_gapic"],
+    importpath = "cloud.google.com/go/compute/apiv1",
+)
+
+# Open Source Packages
+go_gapic_assembly_pkg(
```

Comment:
Just as a sanity check - we do not need a transport parameter for assembly, do we (mayb e neded to generated proper build files and stuff)?

Reply:
Nope we do not need transport parameter for assembly. The `transport` param just toggles which transport interface implementations to generate. REST and/or gRPC. The code all goes into the same files, etc.